Source/JavaScriptCore/runtime/JSONObject.cpp

```diff
@@ -751,10 +829,28 @@ inline void FastStringifier::recordBufferFull()
     recordFailure("buffer full"_s);
 }
 
-inline unsigned FastStringifier::remainingCapacity() const
+ALWAYS_INLINE bool FastStringifier::hasRemainingCapacity(unsigned size)
 {
     ASSERT(!haveFailure());
-    return std::size(m_buffer) - m_length;
+    ASSERT(size > 0);
+    unsigned remainingCapacity = m_useableEnd - m_length;
+    if (size <= remainingCapacity)
+        return true;
+    return hasRemainingCapacitySlow(size);
+}
+
+bool FastStringifier::hasRemainingCapacitySlow(unsigned size)
+{
+    ASSERT(!haveFailure());
+
+    size_t unusedBufferSize = sizeof(m_buffer) - m_length;
```

Comment:
This doesn’t need to be a `size_t` computation.

Reply:
Changed to unsigned.